packages/remark-parse/lib/tokenize/list.js

```diff
@@ -432,7 +432,8 @@ function normalListItem(ctx, value, position) {
 
   lines = value.split(C_NEWLINE);
 
-  trimmedLines = removeIndent(value, getIndent(max).indent).split(C_NEWLINE);
+  var maxIndent = getIndent(max);
```

Comment:
could you move the `var` declaration to the top? So `var maxIndent` above, and `maxIndent = getIndent(max)` here?